common/config_parser/config_parser.cpp

```diff
@@ -426,6 +426,12 @@ ExpectedBool MenderConfigFromFile::ValidateConfig() {
 	if (!server_conf) {
 		return server_conf;
 	}
+
+	if (!dbus_enabled) {
+		log::Error("Turning DBus off is no longer supported");
+		return false;
+	}
```

Comment:
Now that I think of it, why even keep this option? We can just remove it as well. We are committed to DBus at this point, it's extremely unlikely that this will ever be an allowed configuration option.

Reply:
I did exactly that. Let me know if you disagree.

Reply:
I agree. Good thinking.